datahub-web-react/src/app/entityV2/shared/components/styled/DeprecationIcon.tsx

```diff
@@ -32,10 +34,8 @@ const DeprecatedTitle = styled(Typography.Text)`
 const DeprecatedSubTitle = styled(Typography.Text)`
     display: block;
     margin-bottom: 5px;
+    font-size: 12px;
     color: ${REDESIGN_COLORS.TEXT_HEADING};
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
```

Comment:
so we don't want this getting truncated anymore? maybe we're just letting it wrap/show the full subtitle now? this looks like it's only to show deprecated replacement

Reply:
We are using compact markdown viewer which allows the user to click "show more" to expand to see the entire note

Reply:
right yeah that makes sense, i just think this section is for the section above the compact markdown viewer with the note where we show the replacement of the deprecated asset. not positive why this overflow was necessary though there tbh

Reply:
oh yeah it's used for both the replacement AND the thing here. 

I don't really understand why overflow hidden was required here